runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkBatchPortablePipelineTranslator.java

```diff
@@ -115,7 +115,8 @@
    * Creates a batch translation context. The resulting Flink execution dag will live in a new
    * {@link ExecutionEnvironment}.
    */
-  public static BatchTranslationContext createTranslationContext(JobInfo jobInfo) {
+  public static BatchTranslationContext createTranslationContext(JobInfo jobInfo,
```

Comment:
Nit: These aren't arbitrary staging files as far as Flink is concerned. They're jars that will be added to TaskManager classpaths. I think that using more precise language here (instead of `filesToStage`) should make the intent clearer.

Reply:
How about beamDriverFiles?

Reply:
That sounds good.